### ISSUE TITLE
Questions About Training

### ISSUE THREAD

**Issue by ZihCode** [xiph/rnnoise#208]
After training, the "sound" part of my test was also severely weakened. Is this because I selected an inappropriate data set? How should I train to accurately recognize noise and human voice? Do I need to modify the loss function? Can you provide the raw duration of your training set and the settings of your parameters? I'm in 4H noise and 4H pure human voice files at 4000000 COUNT and 128 batch_size. With 120 epoch and my final val_loss is around 2. In this case, both sound and noise are suppressed to a large extent.Is there anything inappropriate between my data set and the parameters I selected？
I would be very appreciate for your help. Thx!
